RELEASE/scripts/autoscend.ash

```diff
@@ -3146,7 +3146,7 @@ int speculative_pool_skill()
 	{
 		expectPool += my_inebriety();
 	}
-	if((have_effect($effect[Chalky Hand]) > 0) || (item_amount($item[Handful of Hand Chalk]) > 0))
+	if(auto_my_path() != "G-Lover" && (have_effect($effect[Chalky Hand]) > 0 || item_amount($item[Handful of Hand Chalk]) > 0))
```

Comment:
you should use auto_is_valid(item it)

Reply:
it is not an IOTM, can it ever not be valid?

Reply:
ok yea. auto_is_valid does indeed check glover and bees and unrestricted. so it is perfect here

Reply:
change made